src/dlgIRC.cpp

```diff
@@ -609,7 +609,9 @@ void dlgIRC::slot_receiveMessage(IrcMessage* message)
                 if (!textToLua.isEmpty()) {
                     const QString from = message->nick();
                     const QString to = getMessageTarget(message, buffer->title());
-                    mpHost->postIrcMessage(from, to, textToLua);
+                    if (mpHost) {
```

Comment:
Why not just combine the `mpHost` check as an AND with the other one within the condition of line 609?

Reply:
Added, though it's important to focus on things that actually matter. It helps PRs move quicker.